StudCity.Frontend/src/UI/fields/PasswordTextField.jsx

```diff
@@ -21,42 +32,50 @@ function PasswordTextField({ className = "", placeholder, required, setDisabled,
 				setDisabled(true);
 			}
 
-			if (evnt.target.value.length <= 8 || confirmationValue.length <= 8)
-				setDisabled(true);
+			if (evnt.target.value.length <= 8 || confirmationValue.length <= 8) setDisabled(true);
 
-			setPasswordError(errMsg);
+			setPasswordError(t(errMsg));
 		}
-	}
+	};
 
 	return (
-		<div className={`relative text-[${primaryGray}] flex flex-col`} >
+		<div className={`relative text-[${primaryGray}] flex flex-col`}>
 			{!hidden ? (
-				<svg onClick={() => setHidden(!hidden)}
+				<svg
```

Comment:
why don't we put such svg's into assets folder?
and then we could use and style such svg's in following format:
import { ReactComponent as YourSvg } from 'assets/svgs/your-svg.svg';

Reply:
This is a one-time case, and we need to pass various properties to the component